src/app/features/create/components/hydrate-new-vehicle-record/hydrate-new-vehicle-record.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { GlobalError } from '@core/components/global-error/global-error.interface';
+import { GlobalErrorService } from '@core/components/global-error/global-error.service';
+import { Axle, VehicleTechRecordModel } from '@models/vehicle-tech-record.model';
+import { Actions, ofType } from '@ngrx/effects';
+import { Store } from '@ngrx/store';
+import { TechnicalRecordService } from '@services/technical-record/technical-record.service';
+import { createVehicleRecord, createVehicleRecordSuccess } from '@store/technical-records';
+import { TechnicalRecordServiceState } from '@store/technical-records/reducers/technical-record-service.reducer';
+import { take, Observable } from 'rxjs';
+
+@Component({
```

Comment:
Is this component representing a page that allows you to fill in details for a new record?

Reply:
The rest of the data correct

Reply:
HydrateNewVehicleRecordComponent seems like the name for an action/function that modifies some kind of data object more than something representing a page, just wondering if it could be named differently?

Reply:
Will update when a name comes to us but non blocking :D